app/src/main/res/anim/nav_default_enter_anim.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Kiwix Android
+  ~ Copyright (c) 2020 Kiwix <android.kiwix.org>
+  ~ This program is free software: you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with this program. If not, see <http://www.gnu.org/licenses/>.
+  ~
+  -->
+
+<set xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
My find usages says these animations are unused

Reply:
They are not referenced anywhere but overrides the default animations for navigations [1]. Not the slickest solution but I couldn't find anything more clean and simple.

[1] https://stackoverflow.com/questions/50482095/how-do-i-define-default-animations-for-navigation-actions

Reply:
I don't like the resource overriding that much but I guess it will have to do. I guess as long as they are not reported unused by lint?